public/usage-examples/graphics/option_rotate_bmp-1-example-oop.cs

```diff
@@ -0,0 +1,56 @@
+﻿using SplashKitSDK;
+
+namespace OptionRotateBmpExample
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            // Set the frame rate to 60 frames per second
+            const int fps = 60;
+
+            // Open a window with dimensions 800 x 600
+            SplashKit.OpenWindow("Usage Example - Option Rotate Bmp", 800, 600);
```

Comment:
Try to instantiate the window as an object so you can make calls directly to it rather than using global references to be more inline with OOP practices. 
```suggestion
            Window window = OpenWindow("Usage Example - Option Rotate Bmp", 800, 600);
```